### ISSUE TITLE
GAS balance incorrect. negative claimable GAS

### ISSUE THREAD

**Issue by arrix** [CityOfZion/neo-python#538]
[TestNet] Progress: 1684947/1684947
### Current behavior

- GAS balance incorrect
- available claims is negative

output of wallet command
```
"percent_synced": 100
...
"script_hash": "ALXncHucqcqrM3GocHrc7GkRToWN3zXyyV",
"frozen": false,
"votes": [],
"balances": {
    "0x602c79718b16e442de58778e148d0b1084e3b2dffd5de6b7b16cee7969282de7": "510.0"
}
...
"claims": {
    "available": "-4999994.04742",
    "unavailable": "0.0"
}
```
### Expected behavior

- GAS balance should be 0
- claimable GAS in the wallet should be a positive number

See https://neoscan-testnet.io/address/ALXncHucqcqrM3GocHrc7GkRToWN3zXyyV
for transaction hisotry
### How to reproduce

- New install of neo-python 0.7.5 using pip
- np-prompt to sync testnet
- import wif (I can provide wif privately)
- tried wallet rebuild using 0.7.5 and 0.7.6 but didn't help

### Your environment

Let us know in what environment you're running into the issue:

- OS: Ubuntu 16.04.5 LTS
- neo-python version: 0.7.6
- Python version: Python 3.6.6

**Comment by arrix** [comment]
Updated to 0.7.7.dev0 (development branch)
Purged my local testnet chain and re-synced. Also rebuilt the wallet.

[TestNet] Progress: 1691020/1691020

```
            "version": 0,
            "script_hash": "ALXncHucqcqrM3GocHrc7GkRToWN3zXyyV",
            "frozen": false,
            "votes": [],
            "balances": {
                "0x602c79718b16e442de58778e148d0b1084e3b2dffd5de6b7b16cee7969282de7": "510.0"
            },

...
    "claims": {
        "available": "2000011.22289",
        "unavailable": "0.0"
    }
```

The numbers are still incorrect.

The three addresses in the wallet are
ARBnLVCAMcZeDZnvQSfn9q5zZ7eEyQu9wi
ALXncHucqcqrM3GocHrc7GkRToWN3zXyyV
AKe5QcddN31d8BDjn29CEoFoi2jb3fzvJ9

**Comment by localhuman** [comment]
I'm also seeing similar issues.

**Comment by tomascarvalho** [comment]
+1

**Comment by ixje** [comment]
closing due to inactivity and issue likely to be solved by #611
